spec/requests/mock/authentication_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Mock
```

Comment:
hi @smoothcontract I've refactored the mock controllers a bit so they are no longer being dynamic defined. Instead, each controller is defined at the top of its relevant rspec (nb: class names must be unique). Is this approach cleaner?

Reply:
Yes! I like that much better, good stuff 👍